### ISSUE TITLE
Request to enable FTB Quest pause

### ISSUE THREAD

**Issue by nageih** [FTBTeam/FTB-Mods-Issues#1224]
### Modpack

FTB NeoTech

### Is your suggestion related to a problem? Please describe.

****
I got shot by a skeleton while reading ftbquest, which annoyed the hell out of me
I think it could be enabled and it would improve the gaming experience
![PixPin_2024-06-15_11-32-39.png](https://s2.loli.net/2024/06/15/PCidwDp4q7mJSMH.png)

**Comment by nageih** [comment]
I think it should be in FTB modpack issues，because it's a complete feature
,and just need modpack author enable it@Saereth

**Comment by desht** [comment]
Yeah the feature is already in FTB Quests.  I think the request here is to have it enabled by default in the modpack.
